src/neuroconv/datainterfaces/ophys/miniscope/miniscopeconverter.py

```diff
@@ -50,24 +46,27 @@ def __init__(self, folder_path: FolderPathType, verbose: bool = True):
         """
         self.verbose = verbose
         self.data_interface_objects = dict(
-            Miniscope=MiniscopeImagingInterface(folder_path=folder_path),
-            BehavCam=MiniscopeBehaviorInterface(folder_path=folder_path),
+            MiniscopeImaging=MiniscopeImagingInterface(folder_path=folder_path),
+            MiniscopeBehavCam=MiniscopeBehaviorInterface(folder_path=folder_path),
         )
 
+    def get_conversion_options_schema(self) -> dict:
+        return self.data_interface_objects["MiniscopeImaging"].get_conversion_options_schema()
+
     def add_to_nwbfile(
         self,
         nwbfile: NWBFile,
         metadata,
-        stub_test: bool = False,
-        stub_frames: int = 100,
+        conversion_options: Optional[dict] = None,
+        **kwargs,
```

Comment:
This works, but I know how this looks.. The issue when you're defining conversion_options for a standalone converter the `conversion_options` are passed as a dict, but when you're using it inside another converter the `conversion_options` are passed as `**conversion_options`. I will try to come up with a nicer solution but this is the best I have so far without touching `NWBConverter`.

https://github.com/catalystneuro/neuroconv/blob/cc15563a82cdc8dc6904d40372ee47123ad63737/src/neuroconv/nwbconverter.py#L111-L116

Reply:
Ahhh nice catch, I see the problem

That would probably be a fine minimal alteration of the core NWBConverter, instead of `**conversion_options.get(...)` -> `conversion_options=conversion_options.get(interface_name, dict())`

Then no need for `kwargs` in the Miniscope case

I think the new `add_to_nwbfile` syntax might make that possible where before it wasn't as easy to pass

Reply:
Thanks @CodyCBakerPhD, the tests are working for me locally. we'll see how it goes for the CI